src/components/ec-timer/ec-timer.story.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+import EcTimer from './ec-timer';
+
+<Meta title="Timer" component={EcTimer} />
+
+# EcTimer
+
+This is our `EcTimer` component: 
```

Comment:
We should avoid pronouns in technical documentation like our, you, we. 

Instead of we need more descriptive sentence, Saying that EcTimer is our EcTimer component is not helping anybody and sentence like that can be removed. What this component does?

"Simple countdown component with a visual indicator"

Reply:
You are right, but this was a simple example of how to do it, the Ux team will be the people in charge of doing this doc pages, that's why was so simple, I'll fix it anyway